src/herder/HerderSCPDriver.cpp

```diff
@@ -711,22 +723,82 @@ HerderSCPDriver::combineCandidates(uint64_t slotIndex,
                 {
                     highestTxSet = cTxSet;
                     highest = sv.txSetHash;
+                    if (candidateCloseTimes.size() == 0)
+                    {
+                        throw std::logic_error(
+                            "Missing candidate closeTimes list");
+                    }
+                    highestTxSetIndex =
+                        make_optional<size_t>(candidateCloseTimes.size() - 1);
+                    if (candidateCloseTimes[*highestTxSetIndex] != sv.closeTime)
```

Comment:
this becomes irrelevant if you use one data structure instead of many

Reply:
It's been removed, although a different way (see comment above).